### ISSUE TITLE
how can i received callback when pip was closed

### ISSUE THREAD

**Issue by haohaozaici** [android/media-samples#19]
when pip window was closed, my app still playing the video. i'm unwilling to stop video when onStop called, because maybe it just restored fullscreen. So i hope receive closed callback, i can really destroy activity and release my MediaPlayer.

**Comment by anonym24** [comment]
> don't finish the activity

are you ok? don't answer to questions if you have no idea what are you about

**Comment by anonym24** [comment]
yeah, it seems close button on the PIP just stops the activity but doesn't destroy it,  I would expect it be finished as back button behavior

**Comment by Dvl-es** [comment]
@haohaozaici 
You can do something like this:

` @Override
  public void onPictureInPictureModeChanged(boolean isInPictureInPictureMode) {
    super.onPictureInPictureModeChanged(isInPictureInPictureMode);

    if (!isInPictureInPictureMode) { 
// Pip is closed here
   }
  }`
